src/Microsoft.IdentityModel.Tokens/ECDsaSecurityKey.cs

```diff
@@ -88,5 +89,25 @@ public override SignatureProvider GetSignatureProvider(string algorithm, bool ve
             else
                 return CryptoProviderFactory.CreateForSigning(this, algorithm);
         }
+
+        public override bool IsSupportedAlgorithm(string algorithm)
+        {
+            if (string.IsNullOrEmpty(algorithm))
```

Comment:
Is there any reason to tolerate null/empty strings? Throwing would probably be more appropriate.